persistence-api/pom.xml

```diff
@@ -99,9 +126,15 @@
             <Bundle-Name>Persistence</Bundle-Name>
             <Bundle-Description>DB component for project</Bundle-Description>
             <Bundle-SymbolicName>io.stargate.db</Bundle-SymbolicName>
-            <Import-Package>org.osgi.framework</Import-Package>
+            <Bundle-Activator>io.stargate.db.DbActivator</Bundle-Activator>
```

Comment:
Why this is called `DdActivator` and not for example `CDCActivator`?

Reply:
Because it's the activator for the `persistence-api` module.